docs/dev/update.md

```diff
@@ -0,0 +1,165 @@
+# Update
+
+This document describes steps required to update the Sumo Logic Kubernetes Collection Helm Operator.
+
+## Components images update
+
+1. Build and certify components container images, make steps from [this][container_cerification] list.
+
+1. Save output of `make verify` from [sumologic-openshift-images][sumologic-openshift-images] to the file, e.g.
+
+   ```bash
+   make verify > images.txt
+   ```
+
+1. Check the content of `images.txt` and make sure that all images are available - no `MISSING` entries in the `images.txt`.
+
+1. Check the content of `images.txt` and remove duplicated entries for single component (multiple versions of container images for single component).
```

Comment:
We could provide combination of `uniq`, `sort` and `grep` but not sure if its worth to do it

Reply:
The content in images.txt is sorted, it is not trivial command to find 4 entries when 2 entries are ok 🤔 
I would leave it as it is.